BackendApp/routeProject.md

```diff
@@ -3,17 +3,23 @@
 - GET /shelters - lista schronisk
 - GET/PUT/DELETE /shelters/id - zawartość schroniska o numerze id
 - POST /shelters/add - dodaje schronisko
+
+- GET /shelters/id/permissions/ - permissions do danego schroniska
+- POST /shelters/id/permissions/add - dodanie permission do danego schroniska
+- GET/PUT/DELETE /shelters/id/permissions/permission_id - edycja wybranego permission do schroniska
+
+- GET /shelters/id/requests - zwraca wszytskie reguesty wysłane do schroniska
+- GET/DELETE /shelters/id/requests/request_id - dostęp do requesta o permission (nie można go edytować!)
 - POST /shelters/id/make-request - pozwala użytkownikowi zrobić permission request
-- POST /shelters/id/resolve-request - pozwala uprawnionemu użytkownikowi zaakceptować permission request
+- POST /shelters/id/requests/request_id/resolve-request - pozwala uprawnionemu użytkownikowi zaakceptować permission request
```

Comment:
Same as in above comments, requests should probably have their own endpoint `/permission-requests` or something